less/pages/thunderbird.less

```diff
@@ -1,4 +1,8 @@
 .thunderbird {
+  &.thank-you-page,
+  &.share-page {
+    display: block;
```

Comment:
Thunderbird uses the old way of displaying background. I'm fine with it, but needs to be reset here.